stalled.js

```diff
@@ -1,16 +1,36 @@
-const async = require('async');
-const { storage, versioning } = require('arsenal');
+const assert = require('assert');
 const { Logger } = require('werelogs');
 const ZenkoClient = require('zenkoclient');
 
-const { MongoClientInterface } = storage.metadata.mongoclient;
-const { encode } = versioning.VersionID;
+const {
+    MongoClientInterfaceStalled,
+} = require('./StalledRetry/MongoClientInterfaceStalled');
+
+const {
+    BackpressureCursor,
```

Comment:
I believe the class is now named `RateLimitingCursor`, `BackpressureCursor ` would be undefined if imported this way.